docs/index.rst

```diff
@@ -19,52 +19,48 @@ user-defined types among other features.
 With Solidity you can create contracts for uses such as voting, crowdfunding, blind auctions,
 and multi-signature wallets.
 
-.. note::
-    The best way to try out Solidity right now is using
-    `Remix <https://remix.ethereum.org/>`_
-    (it can take a while to load, please be patient). Remix is a web browser
-    based IDE that allows you to write Solidity smart contracts, then deploy
-    and run the smart contracts.
+Language Documentation
+----------------------
+
+If you are new to the concept of smart contracts we recommend you start with
+:ref:`an example smart contract <simple-smart-contract>` written
+in Solidity. When you are ready for more detail, we recommend you read the
+:doc:`"Solidity by Example" <solidity-by-example>` and :doc:`"Solidity in Depth" <solidity-in-depth>` sections to learn the core concepts of the language, and before working on production code, make sure you read the :ref:`security_considerations` section.
+
+For further reading, try :ref:`the basics of blockchains <blockchain-basics>`
+and details of the :ref:`Ethereum Virtual Machine <the-ethereum-virtual-machine>`.
+
+.. hint::
+  You can always try out code examples in your browser with the
+  `Remix IDE <https://remix.ethereum.org>`_. Remix is a web browser based IDE
+  that allows you to write Solidity smart contracts, then deploy and run the
+  smart contracts. It can take a while to load, so please be patient.
 
 .. warning::
-    Since software is written by humans, it can have bugs. Thus, also
-    smart contracts should be created following well-known best-practices in
-    software development. This includes code review, testing, audits and correctness proofs.
-    Also note that users are sometimes more confident in code than its authors.
-    Finally, blockchains have their own things to watch out for, so please take
-    a look at the section :ref:`security_considerations`.
+    Humans write software, and it can have bugs. You should follow established
+    software development best-practices when writing your smart contracts, this
+    includes code review, testing, audits, and correctness proofs. Smart
+    contract users are sometimes more confident with code than their
```

Comment:
Please change the line breaks so that the line does not start with `contract` because that triggers the CI to try to compile this indented block, which of course fails.

Reply:
Ha! Done.